src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Bindings/QueueBindingProvider.cs

```diff
@@ -201,7 +201,7 @@ internal async Task<CloudQueue> GetQueueAsync(QueueAttribute attrResolved)
                 string queueName = attrResolved.QueueName.ToLowerInvariant();
                 QueueClient.ValidateQueueName(queueName);
 
-                return client.GetQueueReference(queueName);
+                return Task.FromResult(client.GetQueueReference(queueName));
```

Comment:
Fixed in latest: https://github.com/Azure/azure-webjobs-sdk/blob/ef265e20d4357f2a82871b63a778c99cb6aac8db/src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Config/QueuesExtensionConfigProvider.cs#L205